kibble.json

```diff
@@ -6,6 +6,7 @@
   "defaultLanguage": "en",
   "languages": {
     "ar": { "code": "ar_LB", "name": "عربى" },
+    "ca": { "code": "ca_ES", "name": "Català" },
```

Comment:
```suggestion
    "ca": { "code": "es_CA", "name": "Català" },
```

Reply:
`ca_ES` is lang_REGION though...

Reply:
You are right! My bad, I thought it was Spanish as spoken in Catalan - but Catalan is a language not a country/region. In which case the future convention would be just `ca`.